heartbeat/heartbeat.py

```diff
@@ -23,5 +23,10 @@
         print("Variable TEMPLATE_ID is missing")
     for base_url in BASE_URL:
         notifications_client = NotificationsAPIClient(API_KEY, base_url=base_url)
-        response = notifications_client.send_email_notification(email_address=EMAIL_ADDRESS, template_id=TEMPLATE_ID)
-        print("Email has been sent!")
+        try:
+            response = notifications_client.send_email_notification(email_address=EMAIL_ADDRESS, template_id=TEMPLATE_ID)
+            print("Email has been sent by {}!".format(base_url))
+        except Exception as e:
+            raise e
```

Comment:
If we get a 400 or something then it won't raise an exception - we should check out the status_code we get back. https://docs.notifications.service.gov.uk/python.html#error-codes
(question: what happens when we have a bad url, like we do now with api-k8s on prod?)

Reply:
A `400` would raise an exception but not `200` you mean, no?